runner/bb.edn

```diff
@@ -31,7 +31,7 @@
                                     "artifact-store" "http://localhost:8001/ping"
                                     "postgres" "tcp://127.0.0.1:5433"
                                     "rabbitmq" "tcp://127.0.0.1:5673"})
-                       (shell "clojure -M:test")
+                       (apply shell "clojure -M:test" *command-line-args*)
```

Comment:
I only added it in the `runner` to ask for your input, but we could add it to each service.

Reply:
Let's add it to both, keep it consistent.

Reply:
Also would be nice to doc it in doc/dev.md

Reply:
will do

Reply:
@lispyclouds added the changes